lib/resque/worker.rb

```diff
@@ -362,7 +362,7 @@ def prune_dead_workers
     # lifecycle on startup.
     def register_worker
       redis.sadd(:workers, self)
-      started!
+      redis.set("worker:#{self}:queues", @queues.join(","))
```

Comment:
What are your thoughts on pipelining these redis calls and the ones in unregister_worker? It seems like that might reduce application measured time quite a bit as well by lowering network call overhead.